src/helper/Site_Letsencrypt.php

```diff
@@ -182,8 +182,8 @@ public function authorize( Array $domains, $wildcard = false ) {
 				$payload = $authorizationChallenge->toArray()['payload'];
 
 				$fs = new \Symfony\Component\Filesystem\Filesystem();
-				$fs->copy( SITE_TEMPLATE_ROOT . '/vhost.d_default_letsencrypt.mustache', EE_CONF_ROOT . '/nginx/vhost.d/default' );
-				$challange_dir = EE_CONF_ROOT . '/nginx/html/.well-known/acme-challenge';
+				$fs->copy( SITE_TEMPLATE_ROOT . '/vhost.d_default_letsencrypt.mustache', EE_ROOT_DIR . '/nginx/vhost.d/default' );
+				$challange_dir = EE_ROOT_DIR . '/nginx/html/.well-known/acme-challenge';
```

Comment:
This should be `EE_ROOT_DIR . 'services/nginx-proxy/html/.well-known/acme-challenge`